settings.gradle.kts

```diff
@@ -38,4 +38,6 @@ includeBuild("plugin")
 includeBuild("xdk")
 includeBuild("manualTests")
```

Comment:
I would argue that your original assessment was correct and we don't need "manualTest" to be a part of the main build. If one needs such a "sanity check", he could trivially say:
    ./gradlew :manualTests:build

Reply:
I'll look for a completely excluded way to lift manual tests out of the build, but still can be invoked somehow from the XVM repo build system. The point of pointing something to a build system is that the build system can (optionally) compile and run it. I'll see what i can do with a more complete detachment.

Reply:
Found a way to do it.